projects/kleva/index.js

```diff
@@ -0,0 +1,21 @@
+const retry = require('async-retry')
+const axios = require("axios");
+const BigNumber = require("bignumber.js");
+const { toUSDTBalances } = require('../helper/balances');
+
+async function fetchLiquidity() {
+  const tvl = await retry(async bail => await axios.get('https://bbs5kn3x0j.execute-api.ap-southeast-1.amazonaws.com/dev/klay/ksInfo/tvl'))
```

Comment:
Hi, can you swap this with calls to klaytn chain for calculating tvl, let me know if you need some help

Reply:
Could you elaborate it in detail? This call heads to AWS Lambda which is connected to Klaytn chain already. What do you mean "swap this with calls to klaytn chain"?

Reply:
@graham01u "AWS Lambda which is connected to Klaytn chain" is this part is opaque and creates dependency on this endpoint working/maintained. But the values are fetched by making the calls against chain rpc from our adapter means we have more control, also would be able to fetch historical data. Let me know if you need some help with setting up the adapter :)